cmake/Modules/HandleOutOfTreeLLVM.cmake

```diff
@@ -0,0 +1,80 @@
+if (NOT DEFINED LLVM_PATH)
```

Comment:
I am not sure that these cmake changes should be included in this patch. Many folks concerned with cmake changes will likely notice that this PR has these changes.

Reply:
That was a careless mistake, failed to keep different development branches separate. 

Fixed.